README.rst

```diff
@@ -78,53 +78,97 @@ To activate the virtual environment run ::
 
     pipenv shell
 
-Preferred Environment
----------------------
 
-Please refer to `Working with Openshift`_.
```

Comment:
Why is this getting removed?

Reply:
@blentz Great question!  It was not removed.  It was moved down since it's not the way most people develop.  It is not located in the `Running in a Production-like Environment` section.